README.md

```diff
@@ -1,3 +1,18 @@
+## ORCA Static Documentation
+
```

Comment:
I would add a sentence or two and point them to our nasa.github.io/cumulus-orca pages.

Reply:
Added information and link to the ORCA documentation pages in the README.